### DISCUSSION TITLE
Durable kv watching consumer

### DISCUSSION THREAD

**Discussion by tekumara** [nats-io/nats.py#498] [Ideas]
hiya is it possible to have a durable consumer watch the kv store?

should I reimplement [watch()](https://github.com/nats-io/nats.py/blob/635fb2e1cb3a4813c2b13db908686e36a508c430/nats/js/kv.py#L383C11-L383C11) or is there a better way?

**Comment by jnmoyne** [comment]
Should be easy to just create a durable on the KV bucket's underlying stream.

**Comment by tekumara** [comment]
Example [here](https://github.com/tekumara/nats-demo/blob/79cb5b788d3a15533ecf238fec7e30fbe77f2898/src/demo/durable_watch.py).
